declarations_site/spotter/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class SearchTask(models.Model):
+    user = models.ForeignKey(User)
+    query = models.CharField("Пошуковий запит", max_length=150)
+    deepsearch = models.BooleanField("Шукати скрізь", default=False)
+    query_params = models.TextField("Опції", blank=True, default="", max_length=500)
+    is_enabled = models.BooleanField("Акнтивний", default=True)
+    is_deleted = models.BooleanField("Видалений", default=False)
+    found_total = models.IntegerField("Знайдено всього", default=0)
+    found_week = models.IntegerField("Знайдено за тиждень", default=0)
+    found_ids = models.TextField("Знайдені документи", blank=True, default="", max_length=int(16e6))
+    last_run = models.DateTimeField("Останній запуск", null=True, blank=True, default=None)
+    create_date = models.DateTimeField("Створений", auto_now_add=True, blank=True)
+    change_date = models.DateTimeField("Змінений", auto_now=True, blank=True)
```

Comment:
That'll change on every change of the object. Thus, each update of the last_run will change change_date as well.